### PR TITLE
Added name functionality

### PR DESCRIPTION
**Pull request by kikutestground**
Fixed all bugs alongside. Let me know if you come over any issue. Feedback is appreciated. Thank you.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -98,7 +98,7 @@ label {
 
 }
 
-.input-box:focus ~ label {
+input:focus+label, input:valid+label {
```
Comment (kishlaykiku) [inline css/style.css:101]: Nice. This was much needed fix.

File: script/script.js [hunk 2]
```diff
@@ -14,6 +14,34 @@ function clearTimeOuts()
...
+    //Get name from input field
+    let player2Name = $("#player2Input").val();
+
+    //Validate and push names
+    if(player2Name != "")
+    {
+        $("#player2Name").text(player2Name);
+        $("#player2Input").val("");
+    }
+}
```
Comment (kishlaykiku) [inline script/script.js:42]: We can shorten these two methods into a single method

File: script/script.js [hunk 3]
```diff
@@ -78,12 +106,20 @@ function diceGame()
...
     diceNumber2 = 0;
 
+    //Resets Player names in the card
+    $("#player1Name").text("Player 1");
+    $("#player2Name").text("Player 2");
+    //Reverts back the css for "#win"
     $("#win").css({"opacity":"0", "transition-duration":"200ms"});
+    //Chnages Roll button text to "Roll"
     $(".btn:first").text("Roll");
+    //Resets the gif to Dice 6 png
```
Comment (kishlaykiku) [inline script/script.js:122]: Thanks for documenting.

File: script/script.js (hunk 2 above)
Comment (kikutestground) [reply script/script.js:42]: Alright I will work on this issue and create another pull request

**Review by kishlaykiku — APPROVED** [review]
All fixed. Good job and thank you.

**Review by kikutestground — COMMENTED** [review]
Thank you for your feedback.

**Pull request merged**